app/controllers/api/databases.php

```diff
@@ -2901,10 +2899,11 @@ function updateAttribute(
                 throw new Exception(Exception::ATTRIBUTE_NOT_AVAILABLE, 'Attribute not available: ' . $oldAttributes[$attributeIndex]['key']);
             }
 
-            $lengths[$i] = null;
-
+            $lengths[$i] = array_key_exists($i, $lengths) ? $lengths[$i] : null;
```

Comment:
Can be a bit simpler:

```suggestion
            $lengths[$i] ??= null
```